app/models/package.rb

```diff
@@ -117,13 +117,22 @@ def merge_fields(new_values)
   end
 
   def update_fields
-    to_hash.with_indifferent_access.slice(
-      :isSelected,
-      :allowEbscoToAddTitles,
-      :visibilityData,
-      :customCoverage,
-      :packageName,
-      :contentType
-    )
+    if isCustom?
+      to_hash.with_indifferent_access.slice(
+        :isSelected,
+        :allowEbscoToAddTitles,
+        :visibilityData,
+        :customCoverage,
+        :packageName,
+        :contentType
+      )
+    else
+      to_hash.with_indifferent_access.slice(
+        :isSelected,
+        :allowEbscoToAddTitles,
+        :visibilityData,
+        :customCoverage
+      )
+    end
```

Comment:
@sadatay if you can do this more elegantly, feel free to add another commit to this PR.